Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```
@@ -3211,26 +3272,6 @@
     rules: ghost-role-information-SyndiCat-rules
     raffle:
       settings: default
```

bombroach isnt going to use this so only add to syndicat
```suggestion
      settings: default
  - type: MeleeWeapon
    altDisarm: false
    angle: 0
    animation: WeaponArcBite
    soundHit:
      path: /Audio/Effects/bite.ogg
    damage:
      types:
        Slash: 6
        Piercing: 6
        Structural: 15
```